package.json

```diff
@@ -19,38 +20,43 @@
   "license": "MPL2",
   "dependencies": {
     "alameda": "^1.0.0",
+    "fxos-mvc": "fxos-eng/mvc#master",
```

Comment:
Usually you list in devDep the dependencies needed to rebuild the project. It looks good like this but it's pretty useless because you can't run the code unless you build it. But that's fine I guess.